### DISCUSSION TITLE
请问你如何在openwrt里面配置蓝灯代理

### DISCUSSION THREAD

**Discussion by xu-jan** [getlantern/lantern-forum-cn#1102] [Q&A]
请问你如何在openwrt里面配置蓝灯代理，现有的插件走的是订阅模式，蓝灯支持这个模式吗，如果支持，应该如何配置，如果不支持，请问在openwrt里面如何配置

**Comment by MaxMKMax** [comment]
https://lantern.io/zh-Hans/faq
@janhsuxu 您好. 您看这个链接可以帮到您吗?

**Comment by xu-jan** [comment]
这个里面没有解决办法

**Comment by MaxMKMax** [comment]
@janhsuxu 明天我会和美国的技术团队开会. 我帮您问看.

**Comment by MaxMKMax** [comment]
@janhsuxu 您好. 我再帮您问着了啊. 请您等一等. 有回复我就马上通知您.

**Comment by MaxMKMax** [comment]
@janhsuxu 
您好. 目前我们还没有 OpenWRT 的软件包，但已经有人提出需求了。
我们的技术团队想问您会在什么情况下使用这个？ （学校、家庭、工作场所）您的需求可以帮助塑造并优先考虑未来的发展。

**Comment by xu-jan** [comment]
使用OpenWRT的情况下，因为蓝灯没有订阅地址，需要自己配置，就需要再用一台机器安装并启动蓝灯，然后在OpenWRT上面配置，这样的话使用不方便，如果有openWRT的包，安装之后直接在上面运行，然后配置代理，这样会方便很多

**Comment by MaxMKMax** [comment]
@janhsuxu 好的. 我帮您反馈您的需求. 谢谢您提供的信息.

**Comment by MaxMKMax** [comment]
@janhsuxu 您好, 我们的技术人员想知道您是在什么情况下会需要openwrt? 是学校, 家庭, 还是工作呢? 
然后一般你们的团队是多少人? 还是就单独一个人?
我们需要做点问卷才能推进开发. 谢谢您啦!

**Comment by xu-jan** [comment]
自己在家用，比如HomeAssistant这个系统就不支持配置代理，只能走软路由代理，这个时候就需要在OpenWRT上面配置代理。还有一个事就是，可以打包一个arm架构的安装包吗，我看目前只有x86架构的安装包

**Comment by MaxMKMax** [comment]
@janxuhsu 
https://wdynhnkxvsdx.slack.com/archives/C054ZBXNVE0/p1738688289259999
您好. 技术团队让我给您提供这个链接. 您看可不可以帮到您. 您再反馈给我知道. 谢谢啦!

**Comment by MaxMKMax** [comment]
@janxuhsu 您看可不可以帮到您. 您再反馈给我知道. 谢谢啦!

**Comment by xu-jan** [comment]
lantern-headless-client的docker模式启动报错，下面是报错信息
<img width="1459" alt="image" src="https://github.com/user-attachments/assets/5986be78-bf58-4bce-a436-807936089565" />
